README.md

```
@@ -1,96 +1,86 @@
-*Add caption explaining your workflow*
+Snap shield workflow: Eye detection → Blink count logic → Song trigger → ESP32 LED control.
 
 For Hardware:
 
 # Schematic & Circuit
-![Circuit](Add your circuit diagram here)
-*Add caption explaining connections*
-
-![Schematic](Add your schematic diagram here)
-*Add caption explaining the schematic*
+https://github.com/a-nuranj/anuranj/blob/main/WhatsApp%20Image%202025-08-09%20at%2005.10.22_b8beafa9.jpg
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)